charts/kms-issuer/values.yaml

```diff
@@ -51,6 +51,14 @@ certManagerApprovalRBAC:
   # -- The namespace where cert-manager service account is deployed
   namespace: cert-manager
 
+# Specifies pod Security Context
+podSecurityContext:
+  runAsNonRoot: true
+
+# Specifies Container Security Context
+containerSecurityContext:
```

Comment:
Can we make this a bit more specific, as it only applies to the manager container, not the kube-rbac-proxy container if enabled.